doc/source/fault-tolerance.rst

```diff
@@ -88,3 +88,87 @@ You can experiment with this behavior by running the following code.
             print(counter)
         except ray.exceptions.RayActorError:
             print('FAILURE')
+
+Actor Checkpointing
+~~~~~~~~~~~~~~~~~~~
+
+It's often important to be able to checkpoint the state of an actor in order to
+recover that state in the event of a failure. This can be done by having the
+actor inherit from the ``ray.actor.Checkpointable`` interface.
```

Comment:
can you link to the ``ray.actor.Checkpointable`` docstring here? 

The docstring should also have self-contained examples for implementation.